app/helpers/tabs/observations_helper.rb

```diff
@@ -5,12 +5,12 @@ module Tabs
   module ObservationsHelper
     # assemble links for "tabset" for show_observation
     # actually a list of links and the interest icons
-    def show_observation_tabs(obs:, user:)
-      [
-        observation_manage_lists_tab(obs, user),
-        *obs_change_tabs(obs)&.reject(&:empty?)
-      ]
-    end
+    # def show_observation_tabs(obs:, user:)
+    #   [
+    #     observation_manage_lists_tab(obs, user),
+    #     *obs_change_tabs(obs)&.reject(&:empty?)
+    #   ]
+    # end
```

Comment:
This can definitely be deleted - these are all now available via icon links in show_obs.

Reply:
Sure.